db/migrate/20160718194412_add_confirmable_to_customer.rb

```diff
@@ -0,0 +1,9 @@
+class AddConfirmableToCustomer < ActiveRecord::Migration
+  def change
+    ## Confirmable
+       add_column :customers, :confirmation_token, :string
+       add_column :customers, :confirmed_at, :datetime
+       add_column :customers, :confirmation_sent_at, :datetime
+       add_column :customers, :unconfirmed_email, :string 
+  end
+end
```

Comment:
I think making the user confirmable might've made a good seperate PR. I don't recall this being a story... forgive me if I'm wrong!